dunwoody_disaster/CharacterFactory.py

```diff
@@ -258,6 +258,16 @@ def Cooper() -> Character:
         char.defense = 1
         char.magicDefense = 1
 
+        # char.weapons = [
+        #     Item.Weapon(name="Keyboard", damage=5, magic=5, stamina=5, magical=True),
+        #     Item.Weapon(name="", damage=5, magic=5, stamina=5, magical=False),
+        #     Item.Weapon(name="", damage=5, magic=5, stamina=5, magical=False)
+        # ]
+
+        # char.defenses = [
+        #     Item.Armor(name="", armorVal=5, staminaCost=5, magicCost=5, magicDefense=0),
+        #     Item.Armor(name="", armorVal=5, staminaCost=5, magicCost=5, magicDefense=5)
+        # ]
```

Comment:
If we aren't using it we can delete it

Reply:
We will use it, there's just no values to put there yet.